commit d6cf4043c66ec438fdd2dbf7bbd420e0a22a8392 (refs/changes/27/2827/1)
Author: Tim Freund <tim@freunds.net>
Date:   2017-10-19T23:56:02-04:00 (2 years ago)
    
    Replace Minimum Requirements (404) link in README.md
    
    The site-and-network-requirements page in the wiki no longer exists.  This
    change replaces the Minimum Requirements link with two links:  one to
    the hardware-selection page and one to the network-layout page.

diff --git a/README.md b/README.md
index 4d810ce..ee6901d 100644
--- a/README.md
+++ b/README.md
@@ -262,7 +262,8 @@ available disks on install. You'll want much more storage if you're working with
 images and instances.
 
 If setting up a Triton DataCenter pilot then you'll want to review
-the [Minimum Requirements](https://docs.joyent.com/private-cloud/install/site-and-network-requirements)
+the [Hardware Selection Requirements](https://docs.joyent.com/private-cloud/install/hardware-selection),
+[Networking Layout Requirements](https://docs.joyent.com/private-cloud/install/network-layout),
 and [Installation Prerequisites](https://docs.joyent.com/private-cloud/install/deployment-planning)
 which include IPMI and at least 10 gigabit Ethernet. The supported hardware
 components for SmartOS are described in the [SmartOS Hardware Requirements](http://wiki.smartos.org/display/DOC/Hardware+Requirements).
